### ISSUE TITLE
Sarafi, in full-screen mode modal doesn't get focus 

### ISSUE THREAD

**Issue by limonte** [sweetalert2/sweetalert2#1265]
## Current behavior

Follow-up from https://github.com/sweetalert2/sweetalert2/pull/1264#issuecomment-432829710

> sweetalert is open from Safari when already in full screen, it doesn't get the focus. So the esc key doesn't get handled by sweetalert and it causes the browser to exit from full screen. 

## Expected behavior

In full-screen mode it should work the same, e.g. focus should be trapped in modal when opened.

**Comment by limonte** [comment]
I can't reproduce this in my High Sierra VM (Safari 11.1.2) which version of Safari you're using @gverni?

**Comment by gverni** [comment]
Hi @limonte sorry for the delay but i didn't have my macbook today. The issue is actually happening only on the test/sandbox.html page on the development localhost server. I tested it on https://sweetalert2.github.io and it works fine. So maybe we can decrease the severity, at least until i find the root cause. 

If you want to reproduce, just execute `npm start` and then put safari in full screen and load (or reload) http://localhost:8080/test/sandbox.html.

**Comment by gverni** [comment]
I've done more investigation. It looks like the issue only appears when, while in full screen mode, you type the url the address bar and press enter. I checked few variables when the issue happens, and it looks like everything is fine: 

```
document.hasFocus() = True
window.getSelection().anchorNode.classList = swal2-content
document.activeElement.classList = swal2-input
```

So I believe this has something to do with how Safari is handling the focus in this specific case. I also tried using a simple html page with an input text and I can reproduce the same issue. I'm not sure if it's worth spending more effort on that. I let you decide @limonte.
